src/Namespaces/Eth.php

```diff
@@ -67,4 +68,20 @@ public function getBalance(string $address, string $defaultBlock = null): Wei
 
         return HexToWei::format($result);
     }
+
+    /**
+     * Returns the number of transactions in a block by its hash.
+     *
+     * @throws ErrorException|TransporterException
+     */
+    public function getBlockTransactionCountByNumber(string $blockHash): string
```

Comment:
Can you check if we don't accept a number, and we need to translate to hash before sending to the blockchain?